pkg/orchestrator/message_handler.go

```diff
@@ -225,7 +255,21 @@ func (m *MessageHandler) OnComputeFailure(ctx context.Context, metrics *telemetr
 	metrics.Latency(ctx, messageHandlerProcessPartDuration, AttrPartUpdateExec)
 
 	// enqueue evaluation to allow the scheduler find other nodes, or mark the job as failed
-	if err = m.enqueueEvaluation(txContext, result.JobID, result.JobType); err != nil {
+	jobExecutions, err := m.store.GetExecutions(ctx, jobstore.GetExecutionsOptions{
```

Comment:
The inflight evaluation deduplication verification for the same Job, it should not do that.

Reply:
Verified.